DirectProgramming/DPC++FPGA/include/README.md

```diff
@@ -2,15 +2,22 @@
 This directory contains utility header libraries optimized for FPGA DPC++ designs. Examples of their usage can be found in ReferenceDesigns and Tutorials.
 
 ## Available Header Libraries
+
+### Utilities
+
 | Filename       | Description
 ---              |---
-| pipe_utils.hpp | Defines utilities for working with pipes, such as PipeArray.
 | constexpr_math.hpp | Defines utilities for statically computing math functions such as Log2.
+| pipe_utils.hpp | Defines utilities for working with pipes, such as PipeArray.
```

Comment:
Since you're monkeying with this file anyway, do you mind getting rid of 'Defines . . .' everywhere?  I've given a few examples.

```suggestion
| pipe_utils.hpp | Utility classes for working with pipes, such as PipeArray.
```